src/cart/crt_context.c

```diff
@@ -293,14 +293,14 @@ crt_rpc_complete(struct crt_rpc_priv *rpc_priv, int rc)
 			cbinfo.cci_rc = rpc_priv->crp_reply_hdr.cch_rc;
 
 		if (cbinfo.cci_rc != 0)
-			RPC_ERROR(rpc_priv, "RPC failed; rc: %d\n",
-				  cbinfo.cci_rc);
+			RPC_ERROR(rpc_priv, "failed, " DF_RC "\n",
+				  DP_RC(cbinfo.cci_rc));
 
 		RPC_TRACE(DB_TRACE, rpc_priv,
-			  "Invoking RPC callback (rank %d tag %d) rc: %d.\n",
+			  "Invoking RPC callback (rank %d tag %d) rc: " DF_RC "\n",
```

Comment:
(style)  line over 80 characters

Reply:
why not just move to next line?   We are very inconsistent in how we handle these issues

Reply:
You mean the line length?  Yes, I could do that.